app/scripts/services/ConfCache.js

```diff
@@ -77,6 +77,7 @@ angular
           .add(20, 'days')
           .format('YYYY-MM-DD HH:mm:ss'),
         eventTimezone: 'America/New_York',
+        registrationTimezone: 'Asia/Bangkok',
```

Comment:
Why is `'Asia/Bangkok'` the default for new conferences?

Reply:
Fixed